tests/VolleyManagement.UnitTests/Services/GameReportService/Comparers/PivotStandingsComparer.cs

```diff
@@ -6,6 +6,26 @@
 
     internal class PivotStandingsComparer : IComparer<PivotStandingsDto>
     {
+        private IComparer<TeamStandingsDto> teamsComparer;
+        private bool gameResultComparer = false;
+
+        public PivotStandingsComparer()
+        {
+            TeamStandingsDtoComparer comparer = new TeamStandingsDtoComparer();
```

Comment:
Why you have to call all these methods? Can you set proper default state in constructor?